packages/sdk-nextjs/lib/workflow-designer/call-get-workflow-api.ts

```diff
@@ -0,0 +1,23 @@
+import { WorkflowData, type WorkflowId } from "../workflow-data";
+import { Output } from "../workflow-engine/core/handlers/get-workflow";
+
+export async function callGetWorkflowApi({
+	host = process.env.NEXT_PUBLIC_VERCEL_URL ?? "localhost:3000",
+	api = "/api/workflow/get-workflow",
+	workflowId,
+}: {
+	api?: string;
+	host?: string;
+	workflowId: WorkflowId;
+}) {
+	const response = await fetch(`http://${host}${api}`, {
```

Comment:
nits:

Since requests can only be sent via `http`, it would be helpful if this could also be specified.
I believe this will be improved in another PR, but I’m leaving a comment just in case.